README.md

```diff
@@ -54,56 +55,51 @@ To see detailed help for the entity extraction command, type:
+
+## Advanced CLI uses
+
+The CLI provides additional flags to tune performance, output location, and verbosity of the tool. Some example use case are provided below.
+
+To use a different entity model, use the --space-model flag. The following example directs the tool to use the multi-language model:
+
 ```shell
-sqlite> .schema
-CREATE TABLE file_report (
-	id INTEGER NOT NULL,
-	path VARCHAR,
-	name VARCHAR,
-	size INTEGER,
-	md5 VARCHAR,
-	sha256 VARCHAR,
-	PRIMARY KEY (id)
-);
-CREATE TABLE message (
-	id INTEGER NOT NULL,
-	pff_identifier INTEGER,
-	processing_start_time DATETIME,
-	processing_end_time DATETIME,
-	file_report_id INTEGER,
-	PRIMARY KEY (id),
-	FOREIGN KEY(file_report_id) REFERENCES file_report (id)
-);
-CREATE TABLE entity (
-	id INTEGER NOT NULL,
-	text VARCHAR,
-	label_ VARCHAR,
-	filepath VARCHAR,
-	message_id INTEGER,
-	file_report_id INTEGER,
-	PRIMARY KEY (id),
-	FOREIGN KEY(message_id) REFERENCES message (id),
-	FOREIGN KEY(file_report_id) REFERENCES file_report (id)
-);
+(venv) user@host:~$ ratom entities -p --spacy-model xx_ent_wiki_sm /path/to/PST-or-mbox-file-or-directory
 ```
 
-The schema contains 3 tables representing file information, message information and entity information.
+To specify the number of jobs that may be run concurrently, use the -j flag. The following example limits the number of concurrent jobs to 2:
 
-In the entity table, text is the entity instance, label\_ is the entity type, filepath is the PST file associated with this entity. Full message and file information for each entity are also available through message_id and file_report_id respectively.
+```shell
+(venv) user@host:~$ ratom entities -p -j 2 /path/to/PST-or-mbox-file-or-directory
+```
+
+To change the name or location used for the sqlite3 output file, use the -o flag. Specifying a directory will result in the automatically named file being written to that path. Specifying a path that includes a filename will force the use of that filename. In the following example, the sqlite3 databse will be named filename.db:
+
+```shell
+(venv) user@host:~$ ratom entities -p -o /path/to/directory/filename.db /path/to/PST-or-mbox-file-or-directory
+```
+
+If a job appears is terminating unexpectedly (or failing to terminate), or if you simply wish to view more detailed output during the run, you can increase the level of output verbosity with the -v flag. Additional v's increase verbosity. In the following example, we have increased verbosity to level 2:
```

Comment:
`If a job appears is terminating unexpectedly...` -> `If a job is terminating unexpectedly...`?

Reply:
Fixed.